packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-glossary.tsx

```diff
@@ -45,7 +49,8 @@ const ContractTypeGlossary = ({ category }: { category: string }) => {
                     {
                         type: 'paragraph',
                         text: localize(
-                            'Your payout is equal to the payout per point multiplied by the difference between the final price and the strike price.'
+                            'Your payout is equal to the {{vanilla_payout_text}} multiplied by the difference between the final price and the strike price.',
+                            { vanilla_payout_text: vanilla_payout_text.toLowerCase() }
```

Comment:
In other cases for variables `<Localize />` was used. Is it okay to use `localize()` here? Asking because I thought, that ability to use variables and apply more flexible style is the main difference between them. Please, ignore if not relevant.

Reply:
There's no difference between them, you can use both `<Localize />` and `localize()` interchangeably. It just make more sense to use the tag version directly inside react markup and use function inside javascript code like here. They both support variables and  style insertion.

Reply:
`<Localize />` purpose is not to pass the variables, it's just used for better semantics.